pyjexl/jexl.py

```diff
@@ -71,19 +72,20 @@ def wrapper(func):
     def parse(self, expression):
         try:
             return Parser(self.config).visit(self.grammar.parse(expression))
-        except ParsimoniousParseError as err:
-            raise ParseError('Could not parse expression: ' + expression) from err
+        except ParsimoniousParseError, err:
```

Comment:
using `raise ParseError('Could not parse expression: ' + expression) from err`does't seem to work on 2.7.
are you suggesting something like this? 
`except ParsimoniousParseError as err:
            raise ParseError('Could not parse expression: ' + expression + str(err))`